### PR TITLE
Feature event accept array

### PR DESCRIPTION
**Pull request by abetomo**
It is a fix of #283.

Please point out if there is something about the following two points.

- I reused the existing `--eventFile`, but should I use another option?
- I adopted a method to execute `node-lambda run` internally, but is it better to do another way?

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -95,6 +98,47 @@ Lambda.prototype._runHandler = (handler, event, program, context) => {
   handler(event, context, callback)
 }
 
+Lambda.prototype._runMultipleHandlers = (events) => {
+  console.log(`!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+The event received by AWS Lambda is basically Object.
+For \`node-lambda\`, if event.json is an array,
+constit is treated as multiple events and run multiple times.
```
Comment (DeviaVir) [inline lib/main.js]: Would write it as:
```
Usually you will receive a single Object from AWS Lambda.
We added support for event.json to contain an array,
so you can easily test run multiple events.
```

File: lib/main.js [hunk 2]
```diff
@@ -95,6 +98,47 @@ Lambda.prototype._runHandler = (handler, event, program, context) => {
...
+`)
+
+  const _argv = process.argv
+  const eventFileOptionIndex = (() => {
+    const index = _argv.indexOf('-j')
+    if (index >= 0) return index
+    return _argv.indexOf('--eventFile')
+  })()
+
+  // In order to reproduce the movement of callbackWaitsForEmptyEventLoop,
```
Comment (DeviaVir) [inline lib/main.js]: `the movement` => `the logic`?

File: lib/main.js (hunk 1 above)
Comment (abetomo) [inline lib/main.js]: Thank you!

File: lib/main.js [hunk 3]
```diff
@@ -64,7 +64,10 @@ Lambda.prototype.run = function (program) {
   const event = require(path.join(process.cwd(), program.eventFile))
   const context = require(path.join(process.cwd(), program.contextFile))
 
-  this._runHandler(handler, event, program, context)
+  if (Object.prototype.toString.call(event) !== '[object Array]') {
```
Comment (demian85) [inline lib/main.js]: Why not `Array.isArray()`?

**Comment by demian85** [comment]
Are those events run in order? Would be awesome!

File: lib/main.js (hunk 3 above)
Comment (abetomo) [inline lib/main.js]: Thank you! I'll try!

**Comment by abetomo** [comment]
@demian85 
Since it is asynchronous processing, the order of execution is not guaranteed.
Do you want to modify it to run in order?

**Comment by demian85** [comment]
Yes I can do that later, don't have time today. Perhaps tomorrow or this weekend. Thanks.

**Comment by abetomo** [comment]
@demian85 
I modified to synchronous processing and opened the PR(#313).

**Pull request merged**